permissions/plugin-protecode-sc.yml

```
@@ -4,8 +4,7 @@ github: "jenkinsci/protecode-sc-plugin"
 issues:
   - jira: '21659'  # protecode-sc-plugin
 paths:
-  - "com/synopsys/protecode-sc"
+  - "com/blackducksoftware/protecode-sc"
```

```suggestion
  - "com/blackducksoftware/protecode-sc"
```
This would cause a breakage for updaters. Revert this.
While you can change your group freely for Java classes, revert it in the pom.xml as well, if changed.